scss/content/_typography.scss

```diff
@@ -181,7 +181,8 @@ ul {
 }
 
 // Heading group
-hgroup {
+hgroup,
+.headings {
```

Comment:
you'll want to wrap this addition in the $enable-classes var so that it doesn't show up in the classless version.

Reply:
Thanks, doing it now